swagger.yaml

```diff
@@ -1314,6 +1314,141 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  "/usersSkills":
+    get:
+      operationId: listUsersWithSkills
+      summary: List Users with skills
+      description: Lists Users with skills
+      tags:
+        - UsersWithSkills
```

Comment:
Maybe just UsersSkills is better for tag?

Reply:
If update here then update for all tags

Reply:
I would be consistant and use UserSkills as the name of the entity schema and the endpoint